package.json

```diff
@@ -8,7 +8,10 @@
     "clean": "rm -rf dist/*",
     "start": "set DEBUG=http:* && node ./dist/index.js",
     "start:dev": "set DEBUG=http:* && nodemon src/index.js --exec babel-node",
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "test": "set DEBUG=http:* && cross-env NODE_ENV=test nyc --reporter=html --reporter=text mocha --require @babel/register --timeout 120000 --require @babel/polyfill src/test --exit",
+    "coverage": "nyc report --reporter=text-lcov | coveralls",
```

Comment:
the test script, coverage script and coveralls script have commands doing the same thing(`nyc --reporter`). 
Is it possible to merge show-coverage and coverage script together?

Reply:
It is possible, I will implement it. Thank you

Reply:
done

Reply:
I think the coveralls script is redundant since the coverage script is already calling coveralls